browsermobproxy/client.py

```diff
@@ -97,22 +99,24 @@ def har(self):
         return r.json()
 
 
-    def new_har(self, ref=None, options={}):
+    def new_har(self, ref=None, title=None, options=None):
```

Comment:
Please put `title` at the end of the argument list. If people arent using this as kwargs it is going to break things.